Functions.Templates/Templates/SqlInputBinding-PowerShell/function.json

```diff
@@ -0,0 +1,27 @@
+{
+    "bindings": [
+      {
+        "authLevel": "function",
+        "type": "httpTrigger",
+        "direction": "in",
+        "name": "Request",
+        "methods": [
+          "get",
+          "post"
+        ]
+      },
+      {
+        "type": "http",
+        "direction": "out",
+        "name": "Response"
+      },
+      {
+        "name": "items",
+        "type": "sql",
+        "direction": "in",
+        "commandText": "SELECT * FROM [dbo].[Employees]",
```

Comment:
the readme example (Functions.Templates/Templates/SqlInputBinding-PowerShell/readme.md) uses [dbo].[table1] - if we can't make this a user prompt item let's change it to table1 to be consistent with the readme